tracing-error/src/stack_error.rs

```diff
@@ -0,0 +1,197 @@
+use crate::SpanTrace;
+use std::error::Error;
+use std::fmt::{self, Debug, Display};
+
+struct Erased;
+
+///
+pub struct TracedError<E> {
+    inner: ErrorImpl<E>,
+}
+
+impl<E> From<E> for TracedError<E>
+where
+    E: Error + Send + Sync + 'static,
+{
+    fn from(error: E) -> Self {
+        // SAFETY
+        //
```

Comment:
super nitpicky, but: can we use a consistent style for safety comments? here, we have
```rust
// SAFETY
//
// <rest of comment>
``` 
whereas [on line 53](https://github.com/tokio-rs/tracing/pull/574/files#diff-2638098b229682e4702f7205477eb659R53) we have
```rust
// safety: <rest of comment>
```

i don't really care what style we use for this, as long as it's consistent...and, i would check if other crates in this repo have safety comments, and use whatever style they do.